harness/determined/deploy/gcp/cli.py

```diff
@@ -131,6 +147,17 @@ def handle_up(args: argparse.Namespace) -> None:
     return deploy_gcp("up", args)
 
 
+def handle_dump_master_config_template(args: argparse.Namespace) -> None:
```

Comment:
i think we could drop this and just use https://registry.terraform.io/providers/hashicorp/local/latest/docs/data-sources/file, which would be a lot nicer.